doc/profiles.md

```diff
@@ -298,6 +298,20 @@ This scripts should print a `1 2 3` vector, but only if the `yarp` bindings are
 
 Support for this dependency is enabled by the `ROBOTOLOGY_USES_PYTHON` CMake option.
 
+### System Dependencies
+
+#### Linux
+Install Python and the necessary development files using the following command:
+~~~
+sudo apt-get install python3-dev python3-numpy
```

Comment:
I installed the support written here and successfully installed superbuild repo with 'ROBOTOLOGY_USES_PYTHON' on (ubuntu 20.4) and I tried to import some modules , it works well.

Reply:
Thanks for checking this @HuitongIIT ! To explicitly specify that we are happy if a Pull Request in its current form is merged, in GitHub we use something called "Reviews", and this is the reason I listed you as a "Reviewer" of this PR. In this case I can go forward and merge, but for the future feel free to read a bit more on GitHub Pull Request reviews in https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request#submitting-your-review , thanks!